src/pybind/mgr/orchestrator/module.py

```diff
@@ -352,25 +352,71 @@ def ukn(s):
 
             return HandleCommandResult(stdout=table.get_string())
 
+    @_cli_write_command(
+       'orch osd drivegroups show',
```

Comment:
* `s/show/ls/`
Maybe:
```suggestion
       'orch drive-group ls',
```
or
```suggestion
       'orch osd drive-groups ls',
```
?

Reply:
see https://github.com/ceph/ceph/pull/33381#issuecomment-587531201